infra/modules/cicd/variables.tf

```diff
@@ -41,12 +47,12 @@ variable "labels" {
   default     = {}
 }
 
-variable "run_service_name" {
+variable "repo_owner" {
   type        = string
-  description = "The name of the Cloud Run service that this pipeline will deploy to."
+  description = "Owner of GitHub repository to connect to."
 }
 
-variable "github_repository_url" {
+variable "repo_name" {
   type        = string
```

Comment:
`github_repository_url` was used as a variable input to make it more clear to the end user what the expected value is.  any reason for replace it with 2 separate variables?

Reply:
Passing in through bash script is the reason why.